tests/fixtures/make_sha1_hashes.js

```diff
@@ -3,14 +3,14 @@
 const crypto = require("crypto");
 const stdin = process.openStdin();
 
+const PROMPT = "\nEnter an email address to get the SHA1 hash as it would appear in a HIBP hashset file:";
```

Comment:
Moved the message into a separate variable since it was duplicated in 2 places.
Not sure if this is used directly by [future] tests (since it is IO blocking), or if we should make a bin/ folder at the root level with all our scripts (like this one and /get_hashsets.js).